client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,44 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: #ffffff;
+  color: #ffffff;
+  border: 2px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  padding: none;
+  border:  none;
+  border-collapse: collapse;
```

Comment:
I think you should use `border-collapse: separate` so that the borders will move with the sticky positioned cells. Then to avoid the double border you can remove the top and left borders on the td elements.

Reply:
This didn't quite work; even with no top/left borders there was a single-pixel white space there. But `collapse` seems to work.

Reply:
Done.